worker/spending.go

```diff
@@ -51,25 +51,46 @@ func (w *Worker) initContractSpendingRecorder(flushInterval time.Duration) {
 }
 
 // Record stores the given contract spending record until it gets flushed to the bus.
-func (r *contractSpendingRecorder) Record(rev types.FileContractRevision, cs api.ContractSpending) {
+// func (r *contractSpendingRecorder) Record(rev types.FileContractRevision, cs api.ContractSpending) {
```

Comment:
Commented out code

Reply:
Thanks, was on the fence between RecordV1/V2 or what the weird `recordFn` myself.